src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4751,7 +4751,11 @@ internal static string GetGetVerbText()
 param(
     [Parameter(ValueFromPipeline=$true)]
     [string[]]
-    $verb = '*'
+    $Verb = '*',
+
+    [Parameter(ValueFromPipeline=$true)]
+    [string[]]
+    $Group
```

Comment:
In this case, an enum introduces a new public api.
If the only use of that enum is for parameter validation and tab completion, then `ValidateSet` is might be better than introducing a new api.
If we did use an enum, the parameter doesn't need to be an array, the enum could use `Flags`.